.github/workflows/restart_recovery_suite.yml

```diff
@@ -0,0 +1,113 @@
+name: Restart recovery suite
+on:
+  schedule:
+    - cron: '0 5 * * *'
+  push:
+    branches:
+      - '*' # matches every branch
+  pull_request:
+    branches:
+      - '*' # matches every branch
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    name: Build
+    timeout-minutes: 600
+    runs-on: ubuntu-18.04
+    outputs:
+      err_output: ${{ steps.prepare_err_artifacts.outputs.test}}
+    strategy:
+      fail-fast: false
+      matrix:
+            compiler:
+                - "CONCORD_BFT_CONTAINER_CC=clang CONCORD_BFT_CONTAINER_CXX=clang++"
```

Comment:
Is UDP the default communication layer? Even so, maybe it's worth explicitly stating that the binary is built with the UDP communication module?

Reply:
It's UDP, cause these to flags are set to false.
                              -DBUILD_COMM_TCP_TLS=FALSE \
                              -DBUILD_COMM_TCP_PLAIN=FALSE \